setup.py

```diff
@@ -68,7 +68,7 @@ def _get_dbt_core_version():
     include_package_data=True,
     install_requires=[
         "dbt-core~={}".format(dbt_core_version),
-        "mysql-connector-python>=8.0.0,<8.1",
+        "mysql-connector-python==8.1",
```

Comment:
I'm hesitant to tightly restrict the connector version within the library itself, although your application can certainly pin a specific version. Would a looser restriction like this work?

```suggestion
        "mysql-connector-python>=8.0.0",
```